### ISSUE TITLE
Getting error in google colab and kaggle with simple and minimal code.

### ISSUE THREAD

**Issue by tronizx** [state-spaces/mamba#170]
Hi I'm a beginner in this field. and i trie d mamba mixer simple model with some parameter and tried changing them as well but i got error in google collab.

```
import torch
from mamba_ssm import Mamba
from mamba_ssm.models.mixer_seq_simple import MambaLMHeadModel
from mamba_ssm.models.config_mamba import MambaConfig



mc = MambaConfig()
mc.d_model = 64
mc.n_layer = 64
mc.vocab_size = 500

md = MambaLMHeadModel(config=mc,device='cuda')
md(torch.randn((1,500)).long().to('cuda')).logits.shape 
```

this is minimal code to reproduce in google colab i dont why it is not working it is constantly giving error regardless of any input but suprisingly working with seq len 4 (torch.randn((1,4)), but isnt it too small or i m doing something wrong. please help....

**Comment by sabeaussan** [comment]
Can you describe a bit more what type of error you get ?

**Comment by tronizx** [comment]
> Can you describe a bit more what type of error you get ?

![image](https://github.com/state-spaces/mamba/assets/87223249/49cd5b98-2592-40e7-8a61-4cc56bc7bba4)
this is a screenshot of error...

**Comment by BlenderWang9487** [comment]
> > Can you describe a bit more what type of error you get ?
> 
> ![image](https://private-user-images.githubusercontent.com/87223249/303594066-49cd5b98-2592-40e7-8a61-4cc56bc7bba4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc0NzYwNzAsIm5iZiI6MTcwNzQ3NTc3MCwicGF0aCI6Ii84NzIyMzI0OS8zMDM1OTQwNjYtNDljZDViOTgtMjU5Mi00MGU3LThhNjEtNGNjNTZiYzdiYmE0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA5VDEwNDkzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE5NDMxMGIxMjRkYTQ4ZjI0ZjJlNDdjMWI1ZWMxODVjMjg3MThmYzA5NThmY2ViODI2N2E0YTE0MWE3MTFiOGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QZyPntdv_Ay1gQb-5QtyeuvIDdA_dgUrAkQ9v1ii31s) this is a screenshot of error...

torch.randn follows standard normal distribution and will output negative value

You may not want to pass negative value as input_ids

**Comment by tronizx** [comment]
thanks you so much @BlenderWang9487  its working in google collab but still not in Kaggle environment and i would say it same as before kaggle is quitting environment after runnig the last instruction.

this is my code ->
```
!pip install causal-conv1d>=1.1.0
!pip install mamba-ssm

#cell 2
import torch
import torch.nn as nn
import torch.nn.functional as F
import numpy as np
from tqdm import tqdm

#cell 3
from mamba_ssm import Mamba
from mamba_ssm.models.mixer_seq_simple import MambaLMHeadModel
from mamba_ssm.models.config_mamba import MambaConfig


#cell 4
mc = MambaConfig()
mc.d_model = 64
mc.n_layer = 64
mc.vocab_size = 512

mf = MambaLMHeadModel(config=mc,device='cuda')



#cell 5
mf(torch.randn((1,512)).abs().long().to('cuda'))
```

till cell 4 its is running without any problem but just after cell 5 it some how crash the kaggle environment without any error shown.
like this 
![image](https://github.com/state-spaces/mamba/assets/87223249/b652b0b7-0c98-4d6e-97ee-267809deeba7)

**Comment by notlober** [comment]
try 2xt4 in kaggle, it works
`!pip install mamba-ssm -q

#cell 2
import torch
import torch.nn as nn
import torch.nn.functional as F
import numpy as np
from tqdm import tqdm

#cell 3
from mamba_ssm import Mamba
from mamba_ssm.models.mixer_seq_simple import MambaLMHeadModel
from mamba_ssm.models.config_mamba import MambaConfig


#cell 4
mc = MambaConfig()
mc.d_model = 64
mc.n_layer = 64
mc.vocab_size = 512

mf = MambaLMHeadModel(config=mc,device='cuda')



#cell 5
mf(torch.randn((1,512)).abs().long().to('cuda'))`
output : 
`CausalLMOutput(logits=tensor([[[ 1.0120e-02,  1.1401e-02,  7.8984e-01,  ...,  2.3351e-01,
          -3.6088e-01, -1.4651e-01],
         [ 1.8432e-01,  1.0899e+00,  2.3598e-02,  ...,  1.3865e-01,
           2.6867e-02, -2.5726e-01],
         [ 4.6387e-01,  5.8866e-02,  1.1430e-01,  ..., -3.3296e-02,
           4.0800e-02,  4.3356e-02],
         ...,
         [ 6.5333e-02,  1.6490e-01,  4.4617e-01,  ..., -1.0479e-03,
          -8.9448e-02, -9.1242e-02],
         [ 4.2244e-01, -1.4921e-02,  4.0843e-01,  ...,  6.4403e-03,
          -8.3900e-02, -1.2325e-02],
         [ 2.4284e-01,  1.1364e-01,  8.1052e-02,  ..., -5.7310e-02,
           1.6951e-01, -8.9683e-03]]], device='cuda:0',
       grad_fn=<UnsafeViewBackward0>))`

**Comment by tronizx** [comment]
thank you so much @notlober  and all who helped me. Now it is working perfectly in Kaggle too....it wouldn't be possible for me to figure it out...but all because of your help it is working thanx you so much to you all 🙏
